BUILDING.md

```diff
@@ -1,55 +1,85 @@
-## Building
+# Building
 
-1) Clone the repo to a working directory
+## 1. Clone
 
-2) [CocoaPods](http://cocoapods.org) is used to manage dependencies. Pods are setup easily and are distributed via a ruby gem. Follow the simple instructions on the website to setup. After setup, run the following command from the toplevel directory of Signal-iOS to download the dependencies for Signal-iOS:
+Clone the repo to a working directory
+
+## 2. Depenencies
+
+### tldr;
+
+Build and configure dependencies.
+
+
+```
+make dependencies
+```
+
+### Dependency Details
+
+We have a couple of dependency management tools. We us Carthage for
+managing frameworks, but because some of our dependencies are not yet
+framework compatible, we use Cocoapods to manage the remainder in a
+static library.
+
+2.1) [CocoaPods](http://cocoapods.org) is used to manage the dependencies in our static library. Pods are setup easily and are distributed via a ruby gem. Follow the simple instructions on the website to setup. After setup, run the following command from the toplevel directory of Signal-iOS to download the dependencies for Signal-iOS:
 
 ```
 pod install
 ```
+
 If you are having build issues, first make sure your pods are up to date
+
 ```
 pod update
```

Comment:
I'm not sure we want to recommend a `pod update` when people have build issues.  Shouldn't it just be `pod install`?